frontend/src/components/material/ScheduleEntryLinks.vue

```diff
@@ -8,7 +8,7 @@
     <CategoryChip dense :category="activity.category()" />&thinsp;{{
       $vuetify.breakpoint.smAndUp
         ? `${activity.title}: ${items[0].number}`
-        : items[0].number
+        : shortDescription(items[0])
```

Comment:
I think this would need to be shorter. Maybe just the day and start time?

![image](https://github.com/ecamp/ecamp3/assets/3001985/ae0a60f4-0e1c-4271-8802-0e746c5d7c17)

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/4811/commits/9febee477e783c8d96ffd63765bc04e0f53c679e